src/components/mobile/TopBarSwitch.js

```diff
@@ -8,12 +8,12 @@ import TopBar from '../ui/TopBar'
 const TopBarSwitch = () => (
   <Switch>
     <Route path="/settings"></Route>
-    <Route path="/entry/new">
+    <Route path={['/map/entry/new']}>
```

Comment:
Should we keep these as `="/map/entry/new"`?

Reply:
Yeah, I'll make the change -- I kept a lot of these as arrays when developing since they were easier to change for testing stuff out.